notebook-examples/kge_wikimedia.ipynb

```diff
@@ -0,0 +1,1398 @@
+{
```

Comment:
<p>It should be </p><pre class="ql-syntax" spellcheck="false">pushd dgl-ke;cd python;sudo python3 setup.py install;
</pre><p> </p><p>The master branch now include the inference instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>ok thanks. I have updated the install instructions per your feedback.</p>